pythonforandroid/bootstraps/pygame/__init__.py

```diff
@@ -40,21 +43,14 @@ def run_distribute(self):
             info('Copying python distribution')
             hostpython = sh.Command(self.ctx.hostpython)
             # AND: This *doesn't* need to be in arm env?
-            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir())
+            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir(),
+                    _tail=10, _filterout="^Listing", _critical=True)
```

Comment:
What do these extra arguments do? I thought they would be sh arguments, but couldn't find them.